src/openrct2-ui/windows/Ride.cpp

```diff
@@ -800,8 +800,8 @@ static constexpr const GraphsYAxis window_graphs_y_axi[] = {
 };
 
 static constexpr auto RIDE_G_FORCES_RED_POS_VERTICAL = FIXED_2DP(5, 00);
-static constexpr auto RIDE_G_FORCES_RED_NEG_VERTICAL = -FIXED_2DP(2, 00);
-static constexpr auto RIDE_G_FORCES_RED_LATERAL = FIXED_2DP(2, 80);
+static constexpr auto RIDE_G_FORCES_RED_NEG_VERTICAL = -FIXED_2DP(2, 50);
+static constexpr auto RIDE_G_FORCES_RED_LATERAL = FIXED_2DP(2, 81);
```

Comment:
Don't change this value, instead update the check that determines whether to draw the stats red or white, to match the way penalties are applied.

https://github.com/OpenRCT2/OpenRCT2/blob/e78c29ea5c5b2f301ae8d7d1518c11d01174b516/src/openrct2-ui/windows/Ride.cpp#L5657

The operator here should be changed to `>` instead of `>=`.